server/superdesk/errors.py

```diff
@@ -142,18 +142,23 @@ def __init__(self, code, exception, provider=None):
         provider = provider or {}
         self.provider_name = provider.get('name', 'Unknown provider') if provider else 'Unknown provider'
 
-        if provider.get('notifications', {}).get('on_error', True):
-            exception_msg = str(exception)[-200:]
-            update_notifiers('error',
-                             'Error [%s] on ingest provider {{name}}: %s' % (code, exception_msg),
-                             resource='ingest_providers' if provider else None,
-                             name=self.provider_name,
-                             provider_id=provider.get('_id', ''))
+        if exception:
```

Comment:
`if not exception: return` would be a bit more cleaner IMO or just use one `if exception:`